cpp/src/slice2cpp/Gen.h

```diff
@@ -218,7 +218,7 @@ namespace Slice
             void visitConst(const ConstPtr&) final;
 
         private:
-            MetadataList validate(const SyntaxTreeBasePtr&, MetadataList, const std::string&, int, bool = false);
+            MetadataList validate(const SyntaxTreeBasePtr&, MetadataList, const std::string&, bool = false);
```

Comment:
It's nice to add param names when updating functions. Especially when the param names are non-obvious.

Reply:
You should also add a comment describing what this visitor does; presumably, validate `cpp:...` metadata.